App.js

```diff
@@ -0,0 +1,133 @@
+import { Console } from "@woowacourse/mission-utils";
+
+/**
+ * @enum {string} 계산기 문구 및 구분자 환경변수를 정의하는 열거형입니다.
+ * @property {string} INPUT_PROMPT 덧셈할 문자열을 입력받는 프롬프트 메시지
+ * @property {string} OUTPUT_PROMPT 결과를 출력하는 프롬프트 메시지
+ * @property {string} DEFAULT_SEPARATOR 기본 구분자 (콤마, 콜론)
+ * @property {string} CUSTOM_SEPARATOR 커스텀 구분자(//, \n 사이 문자)
+ */
+const CALCULATOR_VARIABLES ={
+
+    INPUT_PROMPT : '덧셈할 문자열을 입력해주세요.\n', 
+    OUTPUT_PROMPT : '결과 : ',
+    DEFAULT_SEPARATOR : '[,:]',
+    CUSTOM_SEPARATOR : '//(.)\\\\n'
+}
+Object.freeze(CALCULATOR_VARIABLES);
+
+
+
+
+/**
+ * 유저로부터 입력 기능 클래스
+ */
+class InputHandler {
+
+    async getUserInput(){
+
+        try {
+            const userInput = await Console.readLineAsync(CALCULATOR_VARIABLES.INPUT_PROMPT);
+            return userInput;
+        } catch (error) {
+            Console.print('[ERROR]: 입력을 받는 중 문제가 발생했습니다.');
+            return null;
+        }

```

Comment:
의도하신 바인지는 모르겠지만, `getUserInput` 내에서 catch로 에러를 처리하고 다음으로 넘어가게 되면, `isValidSeparator` 에서 에러 처리가 한번 더 되면서 에러 메시지가 두번 출력될 것 같습니다.
따라서 만약에 의도하신 게 아니라면 catch문 아래에서 에러를 출력하는 대신 에러메시지를 바꿔서 다시 에러를 던지는 방법도 있을 것 같아요.

```suggestion
        try {
            const userInput = await Console.readLineAsync(CALCULATOR_VARIABLES.INPUT_PROMPT);
            return userInput;
        } catch (error) {
            throw new Error('[ERROR]: 입력을 받는 중 문제가 발생했습니다.');
        }

```